docs/smart-contracts/databases.md

```diff
@@ -0,0 +1,495 @@
+---
+title: Database model
+description: "Kadena blockchain nodes store information in two different data stores, with a RocksDB key-value store that stores information about nodes, chains, and blocks and a SQLite database that stores information about Pact smart contracts and transactions."
+id: databases
+order: 7
+tags:
+  [
+    'pact',
+    'beginner',
+    'schemas and tables',
+    'schemas',
+    'tables',
+    'pact tutorials',
+  ]
+---
+
+# Database model
+
+Kadena blockchain nodes store information in two different data stores.
+On each Chainweb node, there's a RocksDB key-value stores that stores information about the peer network, chains, and blocks.
```

Comment:
I found this information helpful because a lot of places the distinction is blurred or vague, sometimes chain database is used, sometimes Pact database is used, sometimes database is unqualified and it was hard to sort out what the text was trying to communicate. Those vague references only started to make sense (to me) when I put this rocksdb/sqlite context around it. That's why I included it here.